{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/users/services.py

```diff
@@ -0,0 +1,19 @@
+# Third Party Stuff
+from django.conf import settings
+from django.contrib.auth import get_user_model, authenticate
+
+# {{cookiecutter.main_module}} Stuff
+from {{cookiecutter.main_module}}.base import exceptions as exc
+
+
+def get_and_authenticate_user(email, password):
+    user = authenticate(username=email, password=password)
+    if user is None:
+        raise exc.WrongArguments("Invalid username/password. Please try again!")
+
+    return user
+
+
+def create_user_account(email, password):
```

Comment:
I think, it make sense to add, `first_name` and `last_name` to this as well, which can be optional, as per the model specs.